### ISSUE TITLE
OpenAL in lwjgl not working

### ISSUE THREAD

**Issue by acaly** [rholder/gradle-one-jar#17]
I'm using OpenAL with lwjgl, but OpenAL library could not be found by lwjgl. 

```
org.lwjgl.LWJGLException: Could not locate OpenAL library.
        at org.lwjgl.openal.AL.create(AL.java:151)
        at org.lwjgl.openal.AL.create(AL.java:102)
        at org.lwjgl.openal.AL.create(AL.java:201)
...
```

And I tried to set -Dorg.lwjgl.librarypath to an external path including the binaries, the OpenAL can be loaded properly.

**Comment by acaly** [comment]
I imagine this problem may be fixed easily because the loader only need to add a property (org.lwjgl.librarypath) and set it as java.library.path.

**Comment by acaly** [comment]
The problem seems to be at OpenAL. I'll find help there. Sorry.

**Comment by rholder** [comment]
No worries at all. I'm glad you found another path to getting to a solution. Let me know if you run into any other issues that might be with this project.

**Comment by acaly** [comment]
Thank you. The bug has been fixed in lwjgl/lwjgl#95.
